Test/GTestPluginManager.cpp

```diff
@@ -169,34 +170,48 @@ TEST(TestPluginManagerPathResolution, RespectsAbsolutePath_Config)
 
 TEST(TestPluginManagerPathResolution, RespectsEnvironmentOverride_Plugin)
 {
-    setenv("PLUGIN_PATH", "/tmp/overridden", 1);
+    std::filesystem::path const overriddenPath("/tmp/overridden");
+    std::filesystem::create_directories(overriddenPath);
+
+    auto const exePath = plugin::detail::GetApplicationPath();
+
+    std::filesystem::copy_file(exePath / "plugins" / "libDummyGeneratorPlugin.so", overriddenPath / "some.plugin.so", std::filesystem::copy_options::overwrite_existing);
```

Comment:
Since the code now checks to see if the plugin exists in each of the override paths I need to copy it to the override location for this test to pass.